setup.ts

```diff
@@ -219,6 +219,7 @@ export async function wipeExistingData(url: string): Promise<void> {
       }
       console.log("All existing data has been deleted.");
     }
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Remove the linting disable. Find an acceptable solution. We have a PR test to make sure they are not used.